.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
 - [ ] All new and existing tests passed
 
-<!------------------------->
-<!-- /!\ Core Team Only -->
-<!------------------------->
+#### Documentation
 
-<!-- Uncomment the following for a feature DoD -->
+- [ ] My change introduces changes to the documentation or I have updated the documentation accordingly
 
-<!--
-### Development
-
-- [ ] Should match specs (eg. either the Web UI Kit or any pattern from the WAI — or both…)
-- [ ] Docs added:
-  - including the "Sass" part using `scss-docs` shortcode
-  - in /about/overview/#custom-components if it is a new Orange custom component
-  - in /getting-started/introduction/#components if it is a new Orange custom component that requires JavaScript (and Popper)
-  - in /customize/overview#csps-and-embedded-svgs if it is a new Orange custom component that includes embedded SVGs in our CSS
-  - in /forms/validation/?#supported-elements if it is a new Orange custom component that is a form control
-  - in /forms/overview/ if it is a new Orange custom component that is a form control
-- [ ] (Optional) If this feature is a new component, it probably must appear in Storybook
-- [ ] Check (and fix) RTL version
-- [ ] Run linters
-- [ ] Run compilers
-- [ ] Tests added for JS-side
-- [ ] Run tests
-- [ ] Manually run BrowserStack test
-- [ ] Manually run Percy test
-- [ ] Cross-browser test:
-  - [ ] Chrome
-  - [ ] Firefox (including ESR)
-  - [ ] Edge
-  - [ ] Safari
-  - [ ] iOS Safari
-  - [ ] Chrome & Firefox on Android
-- [ ] Responsive tests: desktop and small screens
-- [ ] Clean up the branch using `rebase -i`
-- [ ] Commited with `feat(…): …` message
-- [ ] Mention it in Migration Guide (if `back-from-v4`): renamed variables, changes in markup requirement,  etc.
-
-### Reviews
+<details>
+  <summary>Detailed info</summary>
+  <ul>
+    <li>including the "Sass" part using <code>scss-docs</code> shortcode</li>
+    <li>in <code>/about/overview/#custom-components</code> if it is a new Orange custom component</li>
+    <li>in <code>/getting-started/introduction/#components</code> if it is a new Orange custom component that requires JavaScript (and Popper)</li>
+    <li>in <code>/customize/overview#csps-and-embedded-svgs</code> if it is a new Orange custom component that includes embedded SVGs in our CSS</li>
+    <li>in <code>/forms/validation/?#supported-elements</code> if it is a new Orange custom component that is a form control</li>
+    <li>in <code>/forms/overview/</code> if it is a new Orange custom component that is a form control</li>
+  </ul>
+</details>
+
+### Checklist (for Core Team only)
 
+- [ ] My change introduces changes to the migration guide
+- [ ] My new component is added in Storybook
+- [ ] Check RTL version
+- [ ] Manually run BrowserStack automatic tests
+- [ ] Manually test browser compatibility with BrowserStack (Chrome, Firefox (+ ESR), Edge, Safari, iOS Safari, Chrome & Android on Android)
 - [ ] Code review
-- [ ] A11y review
 - [ ] Design review
--->
+- [ ] A11y review
```

Comment:
I thought of it for contributors to have a clear state: that there will be a code review, an a11y review and a design review. But I can remove them, I don't mind.

Reply:
Or we could say to add a `-` or `NA` in the checkbox instead of the `x` when not needed ?

Reply:
Or could it be possible to configure GitHub to have 'need design review' and 'need accessibility review' labels automatically added on the creation of each PR?

Reply:
I think that this is fine since folks may not know that we use labels to indicate the state of the PR + Not much to tick a checkbox (might be easier for designer to check it themselves for example).
I think that adding labels on each PR is feasible however, I feel meh, like we'll be lost under labels.

Reply:
> Or could it be possible to configure GitHub to have 'need design review' and 'need accessibility review' labels automatically added on the creation of each PR?

Don't think that's a good idea since it is not true for each PR

Reply:
> when not needed, it won't be checked...
> Or we could say to add a - or NA in the checkbox instead of the x when not needed ?

I will add a note at the beginning (which already exists in comment) to explain that the user needs to transform `- [ ]` into `- (NA)` when not applicable